apps/server/src/modules/pseudonym/repo/pseudonyms.repo.spec.ts

```diff
@@ -118,6 +118,55 @@ describe('PseudonymRepo', () => {
 		});
 	});
 
+	describe('findPseudonymsByUserId', () => {
+		describe('when pseudonym is existing', () => {
+			const setup = async () => {
+				const user1 = userFactory.buildWithId();
+				const user2 = userFactory.buildWithId();
+				const pseudonym1: PseudonymEntity = pseudonymEntityFactory.buildWithId({ userId: user1.id });
+				const pseudonym2: PseudonymEntity = pseudonymEntityFactory.buildWithId({ userId: user1.id });
+				const pseudonym3: PseudonymEntity = pseudonymEntityFactory.buildWithId({ userId: user2.id });
+				const pseudonym4: PseudonymEntity = pseudonymEntityFactory.buildWithId({ userId: user2.id });
+
+				await em.persistAndFlush([pseudonym1, pseudonym2, pseudonym3, pseudonym4]);
+
+				return {
+					user1,
+					pseudonym1,
+					pseudonym2,
+				};
+			};
+
+			it('should return array of pseudonyms', async () => {
+				const { user1, pseudonym1, pseudonym2 } = await setup();
+
+				const result: Pseudonym[] = await repo.findByUserId(user1.id);
+
+				expect(result).toHaveLength(2);
+				expect(result[0].id).toEqual(pseudonym1.id);
+				expect(result[0].userId).toEqual(pseudonym1.userId.toJSON());
+				expect(result[0].pseudonym).toEqual(pseudonym1.pseudonym);
+				expect(result[0].toolId).toEqual(pseudonym1.toolId.toJSON());
+				expect(result[0].createdAt).toEqual(pseudonym1.createdAt);
+				expect(result[0].updatedAt).toEqual(pseudonym1.updatedAt);
+				expect(result[1].id).toEqual(pseudonym2.id);
+				expect(result[1].userId).toEqual(pseudonym2.userId.toJSON());
+				expect(result[1].pseudonym).toEqual(pseudonym2.pseudonym);
+				expect(result[1].toolId).toEqual(pseudonym2.toolId.toJSON());
+				expect(result[1].createdAt).toEqual(pseudonym1.createdAt);
+				expect(result[1].updatedAt).toEqual(pseudonym1.updatedAt);
```

Comment:
normally we use something like this:
https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/modules/files-storage/helper/file-record.spec.ts#L31